src/main/java/com/celements/search/web/classes/WebSearchClassPackage.java

```diff
@@ -12,7 +12,7 @@
 @Component(WebSearchClassPackage.NAME)
 public class WebSearchClassPackage extends AbstractClassPackage {
 
-  public static final String NAME = "webSearch";
+  public static final String NAME = "websearch";
```

Comment:
the class collection certainly is in use, but the class package not yet (different configuration) since I created it for the java WebSearch from this PR